addons/pkg/constants/constants.go

```diff
@@ -235,6 +235,9 @@ const (
 	// ClusterBootstrapManagedSecret is the name for the secrets that are managed by ClusterBootstrapController
 	ClusterBootstrapManagedSecret = "clusterbootstrap-secret"
 
+	// PackageNameLabel is the label key for package name
+	PackageNameLabel = "tkg.tanzu.vmware.com/package-name"
```

Comment:
Maybe we can reuse addons/pkg/types/addons.go PackageNameLabel constant.

Reply:
done